openreview/conference/invitation.py

```diff
@@ -415,12 +417,17 @@ def __init__(self, conference, name, note, start_date, due_date, public):
         content = invitations.meta_review.copy()
 
         readers = ['everyone']
+        regex = conference.get_program_chairs_id()
+        invitees = [conference.get_program_chairs_id()]
+        private_readers = [conference.get_program_chairs_id()]
+
+        if conference.use_area_chairs:
+            regex = conference.get_area_chairs_id(note.number)[:-1] + '[0-9]+'
```

Comment:
What if there are 10 or more ACs?  Too unlikely?

Reply:
+ means one or more characters, so it is unlimited.

Reply:
I meant the [:-1] part only strips off one character.

Reply:
That is removing the 's' from the name, like Area_Chairs --> Area_Chair1